server/models/board.py

```diff
@@ -0,0 +1,127 @@
+"""
+'  Class modeling a box game board
+'
+'  rows:  number of rows in the grid
+'  cols:  number of columns in the grid
+'
+'  edges are maintained in a map of maps with the following structure:
+'    { sourcePoint1: {neighboringPoint1: owner, neighboringPoint2: owner}, sourcePoint2: ... }
+'
+'    where each point is a row-majored tuple of 2 integers describing the row and column of the point.
+'
+'  completed boxes are maintained in a map mapping the top-left corner of the box to the owner of the box
+"""
+
+
+class Board(object):
+
+    def __init__(self, rows, cols):
```

Comment:
Just wanted to say I really like this design, especially with the top-left indication of a filled box.  Nice work man.  😃